### ISSUE TITLE
Review capital of Sri Lanka

### ISSUE THREAD

**Issue by axelboc** [anki-geo/ultimate-geography#436]
While working on #435 (#416), I realised that English Wikipedia lists two capitals for Sri Lanka in the infobox since [May 2020](https://en.wikipedia.org/w/index.php?title=Sri_Lanka&diff=956281856&oldid=956274604) (see also [talk page](https://en.wikipedia.org/wiki/Talk:Sri_Lanka/Archive_11#Capital_city?)):

- **Sri Jayawardenepura Kotte** (legislative)
- **Colombo** (executive and judicial)

Same on [Wikipedia in Sinhala](https://translate.google.com/translate?sl=auto&tl=en&u=https://si.wikipedia.org/wiki/%25E0%25B7%2581%25E0%25B7%258A%25E2%2580%258D%25E0%25B6%25BB%25E0%25B7%2593_%25E0%25B6%25BD%25E0%25B6%2582%25E0%25B6%259A%25E0%25B7%258F%25E0%25B7%2580) (Sri Lanka's main language), but with the capital qualifiers "administrative" and "commercial" respectively.

Therefore, instead of fixing the capital inconsistency in Norwegian ("Colombo", while the English deck has "Sri Jayawardenepura Kotte"), I think we should consider listing both capitals in the _Capital_ field, and reviewing the _Capital info_ field accordingly.

Also, I think the _Capital info_ field should mention that "Sri Jayawardenepura Kotte" is commonly known as "Kotte". I think this will help people who struggle with remembering the full name know which part is most important to remember. 😁

**Comment by josealberto4444** [comment]
What do you think about the following?

Capital
Sri Jayawadenepura Kotte, Colombo

While Sri Jayawadenepura Kotte (commonly known as Kotte) is the administrative and legislative capital, Colombo is the executive and judicial capital.

**Comment by josealberto4444** [comment]
After thinking about this a bit more, I'm leaning to keep only listing Sri Jayawadenepura Kotte as the capital and adding that capital info. Otherwise, maybe we should do it also for Bolivia, Malaysia and every country having more than a city in the infobox (there might be some more).

**Comment by aplaice** [comment]
I fully agree that we should decide on this globally — i.e. not touch Sri Lanka until we've decided on a consistent general policy. I'll add (brief) some comments there.

<hr/>

Assuming that we have two capitals for Sri Lanka, then I think:

> Sri Jayawadenepura Kotte, Colombo
> While Sri Jayawadenepura Kotte (commonly known as Kotte) is the administrative and legislative capital, Colombo is the executive and judicial capital.

works well.
